policyengine_us/parameters/gov/states/ar/tax/income/credits/non-refundable.yaml

```diff
@@ -0,0 +1,11 @@
+description: Arkansas provides these nonrefundable income tax credits.
+values:
+  2022-01-01:
+    - ar_inflation_relief_credit
+metadata:
+  reference:
+    - title: Arkansas Income Tax Return Form 2022 AR1000F
+      href: "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf"
```

Comment:
add reference for all credits and if there is non just the reference for the inflation relief credit + page